src/components/pages/Votation.tsx

```diff
@@ -1,124 +1,188 @@
+
+const subTitlesStyle = {
+  fontStyle: 'normal',
+  fontSize: '16px',
+  fontWeight: 'bold',
+  lineHeight: '150%',
+} as React.CSSProperties;
+
+const h1Style = {
+  fontSize: '1.5em',
+};
 
 const Votation: React.FC = () => {
-  const { id } = useParams<{ id: string }>();
-  const { data, loading, error } = useGetVotationByIdQuery({ variables: { votationId: id } });
-  const votationData = data?.votationById;
+  
+  const { meetingId, votationId } = useParams<{ meetingId: string, votationId: string }>();
+
+  //Get votation data by query
+  const { data: vData, loading: vLoading, error: vError } = useGetVotationByIdQuery({ variables: { votationId: votationId } });
+
+  //Get participants by meeting
+  const { data: mData, loading: mLoading, error: mError } = useGetParticipantsByMeetingQuery({variables: { meetingId: meetingId}})
 
+  //Check if user has voted 
   const { user } = useAuth0();
-  const [hasUserVoted, sethasUserVoted] = useState<boolean>(votationData?.hasVoted?.includes(user) || false);
+  const [hasUserVoted, sethasUserVoted] = useState<boolean>(vData?.votationById?.hasVoted?.includes(user) || false);
 
+  //Register vote
   const [selectedAlternativeId, setSelectedAlternativeId] = useState<string | null>(null);
   const handleSelect = (id: string | null) => setSelectedAlternativeId(id);
-
+  
   const [castVote] = useCastVoteMutation();
   const submitVote = () => {
-    if (selectedAlternativeId !== null) {
+    if (selectedAlternativeId) {
       sethasUserVoted(true);
-      castVote({ variables: { votationId: id, alternativeId: selectedAlternativeId } });
+      castVote({ variables: { votationId: votationId, alternativeId: selectedAlternativeId } });
     }
   };
 
-  if (error) return <Text>Det skjedde noe galt under innlastingen</Text>;
-  if (loading)
-    return (
-      <Center>
-        <Spinner size="xl" m="auto" />
+  //Close votation
+  const [updateVotationStatus] = useChangeVotationStatusMutation();
+
+  if (vLoading || mLoading) {
+    return <>
+        <Box h="57px" w="100vw" bgColor={darkblue}></Box>
+        <Center mt="10vh">
+          <Spinner size="xl" m="auto" />
+        </Center>
+      </>
+  }
+
+  if (vError || mError || vData?.votationById?.id === undefined) {
+    return <>
+      <Box h="57px" w="100vw" bgColor={darkblue}></Box>
+      <Center mt="10vh">
+        <Text>Det skjedde noe galt under innlastingen</Text>
       </Center>
-    );
+    </>
+  }
 
-  const subTitlesStyle = {
-    fontStyle: 'normal',
-    fontSize: '16px',
-    fontWeight: 'bold',
-    lineHeight: '150%',
-  } as React.CSSProperties;
+  //Check if user is admin
+  const participants = mData?.meetingsById?.participants as Array<Participant>;
+  participants.forEach((participant) => console.log(participant))
+  const isUserAdmin = participants?.some((participant) => participant.user?.email === user.email && participant.role === Role.Admin)
+  
+
+  if ( ! mData?.meetingsById?.participants.includes(user)){
```

Comment:
Ettersom vi har tilgangsbegrensning på backend vil man få en vError med meldingen 'Not Authorised' dersom man prøver å hente en votering man ikke er deltaker på. Man vil da vError vil da være true og man vil returnere at det skjedde noe galt og denne vil ikke kjøre. Jeg tror imidlertid denne if-setningen alltid vil være true ettersom mData.meetingsById.participants er en liste med participants, og dermed aldri vil inneholde et user-objekt.

Jeg tenker derfor at denne kan fjernes.

Man kan gjerne kjøre en sjekk på om vError.message === 'Not Authorised!', og dersom det er tilfellet kan man returnere den boksen som returneres under, med at man ikke har tilgang til voteringen. Denne må imidlertid komme før man sjekker vError || mError || vData?.votationById?.id === undefined.